apps/dashboard/app/controllers/dashboard_controller.rb

```diff
@@ -1,10 +1,6 @@
 class DashboardController < ApplicationController
   def index
-    begin
-      @motd = MotdFile.new.formatter
-    rescue StandardError => e
-      flash.now[:alert] = t('dashboard.motd_erb_render_error', error_message: e.message)
-    end
+    @motd = MotdFile.new.formatter
```

Comment:
I'd think we'd want to keep this still. I mean erb rendering failures can still happen right?

Reply:
Yeah it still happens, but my reasoning of removing it was that we're now handling exceptions in the module.

Reply:
Oh i see. Yea I don't think models should know about flashing errors. Mabye they validly can because that functions' available, but it's about separation of concerns.  Let the controller be concerned with flashing errors and dealing with the UI.

Reply:
Gotcha sounds good